modules/ocf_apphost/manifests/systemd_linger.pp

```diff
@@ -0,0 +1,6 @@
+# Declares that the following user is to have a persistent per-user systemd
+define ocf_apphost::systemd_linger($user = $title) {
+  file { "/var/lib/systemd/linger/${user}":
```

Comment:
I think this is acceptable. It's not going to break between Debian releases, and we'll do careful testing on release upgrades anyway.

Agree with your concerns about not using the proper interface, but I think this is better than trying to script that via Puppet.